# DupTree
DupTree is a program for phylogenetic analyses using gene tree parsimony.  That is, given a collection  of binary gene trees, DupTree searches for a species supertree that implies the fewest number of gene  duplication events.


# Download
* Executables for: [Linux](https://iastate.box.com/s/inxu9nltb7fsn8p7gmi01isvheazb85j) , [Mac (Power PC)](https://iastate.box.com/s/9kz587l368cldgynwg1um90drm6vcdr6), [Mac (Intel)](https://iastate.box.com/s/fh8sn53c1h8dhmq1l0mctio5dxdpjo0n), [Windows](https://iastate.box.com/s/1jwky1uu609uvltfpz9yrxk3tn9639qx)
* [Manual](https://iastate.box.com/s/ffow5xjyfl343u70rcwdsrkn8g196q5r)
* [Sample dataset](https://iastate.box.com/s/6jr2txs90zx7vmwmxt1b4j95nrgd12k8)

Source code: DupTree is open source! Please email either Andre Wehe or Mukul Bansal for the source code.

DupTree can be cited as follows:
[DupTree: A program for large-scale phylogenetic analyses using gene tree parsimony](https://academic.oup.com/bioinformatics/article/24/13/1540/238590)
   Andre Wehe, Mukul S. Bansal, J. G. Burleigh, Oliver Eulenstein.
   Bioinformatics, 24(13): 1540-1541, 2008.

NOTE: DupTree is now also available as part of the software package iGTP, which is a graphical program for performing gene tree parsimony analysis under the duplication, duplication+loss, and deep coalescence reconciliation cost models. You can read more about iGTP here.

Contact: Andre Wehe (awehe@iastate.edu) or Mukul Bansal (mukul@csail.mit.edu)
